contents/docs/product-analytics/web-vitals.mdx

```diff
@@ -45,4 +45,28 @@ To create your own dashboard from a template:
     videoDark="https://res.cloudinary.com/dmukukwp6/video/upload/v1725633861/posthog.com/contents/dark.mp4"
     alt="Popover icon" 
     classes="rounded"
-/>
\ No newline at end of file
+/>
+
+## Events captured
+
+The SDK groups available metrics in order to reduce the cost. That means each `$web_vitals` event carries one or more metric result. By default we wait up to 5 seconds (since the page load or last set of metrics was sent) in order to try and catch all four metrics in one event. See the section on configuring web vitals capture below for how to change this behaviour.
+
+Metrics are made available when the [web vitals library](https://github.com/GoogleChrome/web-vitals) triggers them. This is different for different metrics.
+
+[FCP](https://web.dev/articles/fcp) and [LCP](https://web.dev/articles/lcp) are relative to a page being rendered and are made available relatively soon after loading a page. [INP](https://web.dev/articles/inp) and [CLS](https://web.dev/articles/cls) are measures across the lifecycle of a page and don't have a fixed time they become available.
+
+We wait until all allowed metrics have been triggered or `web_vitals_delayed_flush_ms` has passed and then send a single `$web_vitals` event with each of the triggered metrics.
+
+
```

Comment:
I think this just repeats information from the first paragraph or in the section below, less confusing without it.
```suggestion
```